src/geo-choropleth-chart.js

```diff
@@ -193,82 +160,398 @@ dc.geoChoroplethChart = function (parent, chartGroup) {
+                    .scale(s);
+            } else {
+                projection
+                    .translate([width / 2, height / 2])
+                    .scale(Math.min(width, height)  * 2);
+            }
+        } else {
+            var bounds = d3.geo.bounds(features);
+            projection
+                .rotate([-(bounds[0][0] + bounds[1][0]) / 2, 0])
+                .center([0, (bounds[0][1] + bounds[1][1]) / 2])
+                .translate([width / 2, height / 2])
+                .scale(s);
+        }
+    }
+    ```
+    **/
+    _chart.projectionZoom = function (_) {
+        if (!arguments.length) {
+            return _projectionZoom;
+        }
+        _projectionZoom = _;
+        return _chart;
+    };
+
+    /**
+    #### .projectionZoomScale(projectionZoomScale)
+    Set a numerical geo projection scale value to be used when zooming in on a set of features. Default: 0.95.
 
     **/
-    _chart.geoJsons = function () {
-        return _geoJsons;
+    _chart.projectionZoomScale = function (_) {
+        if (!arguments.length) {
+            return _projectionZoomScale;
+        }
+        _projectionZoomScale = _;
+        return _chart;
     };
 
     /**
-    #### .geoPath()
-    Returns the [d3.geo.path](https://github.com/mbostock/d3/wiki/Geo-Paths#path) object used to
-    render the projection and features.  Can be useful for figuring out the bounding box of the
-    feature set and thus a way to calculate scale and translation for the projection.
+    #### .showGraticule(showGraticule)
+    Render graticules. Default: false.
 
     **/
-    _chart.geoPath = function () {
-        return _geoPath;
+    _chart.showGraticule = function (_) {
+        if (!arguments.length) {
+            return _showGraticule;
+        }
+        _showGraticule = _;
+        return _chart;
     };
 
     /**
-    #### .removeGeoJson(name)
-    Remove a GeoJson layer from this chart by name
+    #### .showGraticule(showGraticule)
+    Render world sphere. Default: false.
 
     **/
-    _chart.removeGeoJson = function (name) {
-        var geoJsons = [];
+    _chart.showSphere = function (_) {
+        if (!arguments.length) {
+            return _showSphere;
+        }
+        _showSphere = _;
+        return _chart;
+    };
 
-        for (var i = 0; i < _geoJsons.length; ++i) {
-            var layer = _geoJsons[i];
-            if (layer.name !== name) {
-                geoJsons.push(layer);
-            }
+    /**
+    #### .zoomable(zoomable)
+    Enable a interactive mouse zoom with the current projection.  Uses a modified d3.zoom from:
+    http://www.jasondavies.com/maps/rotate/d3.geo.zoom.js
+
+    **/
+    _chart.zoomable = function (_) {
+        if (!arguments.length) {
+            return _zoomable;
+        }
+        _zoomable = _;
+        return _chart;
+    };
+
+    // PLOT
+    _chart._doRedraw = function () {
+        var data = _layeredData(_chart.data());
+
+        for (var layerName in _layers) {
```

Comment:
It does not since we never bind the layer data using d3 and instead just loop them.  Should be a fairly easy change.